File: eBeertijaBackend/eBeertijaBackend/DTOs/RacunDto.cs
using ebeertijaBackend.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ebeertijaBackend.DTOs
{
    public class RacunDto
    {
        public int Id { get; set; }
        public int Broj { get; set; }
        public string BrojOpis { get; set; }
        public DateTime Datum { get; set; }
        public int? NarudzbaId { get; set; }
        public double UkupnoSaPdvom { get; set; }
        public double UkupnoBezPdva { get; set; }
        public double IznosPdv { get; set; }
        public int StolId { get; set; }
        public string OznakaStola { get; set; }
        public int UserId { get; set; }
        public bool IsStorniran { get; set; }
        public int? StorniraniRacunId { get; set; }
        public string NazivKorisnika { get; set; }
        public ICollection<StavkaRacunaDto> StavkeRacuna { get; set; }
    }
}

File: eBeertijaFrontend/src/Models/RacunDto.ts
import { StavkaRacunaDto } from './StavkaRacunaDto';

export class RacunDto {
    id?: number;
    broj: number;
    brojOpis: string;
    datum?: Date;
    narudzbaId?: number;
    ukupnoSaPdvom?: number;
    ukupnoBezPdva?: number;
    iznosPdv?: number;
    stolId: number;
    oznakaStola: string;
    userId?: number;
    isStorniran: boolean;
    storniraniRacunId?: number;
    nazivKorisnika?: string;
    stavkeRacuna?: StavkaRacunaDto[];
}
